.github/workflows/frontend_tests.yml

```diff
@@ -10,11 +10,33 @@ on:
       - release-*
 
 jobs:
+  generate-job-strategy-matrix:
+    runs-on: ubuntu-22.04
+    outputs:
+      job-strategy-matrix: ${{ steps.generate.outputs.job-strategy-matrix }}
+    steps:
+      - name: Generate job strategy matrix
+        id: generate
+        env:
+          standard-test-runs: 1
+          flakes-test-runs: 50
```

Comment:
I'm worried that 50 runs will put too much strain on our CI queue. What's your plan for evaluating the impact of this PR on CI wait times? Should we start small and scale up?

Reply:
The oppia org has a GitHub Team plan. This means that we can run 60 jobs concurrently. Given that we merge 3 PR's a day on average. I don't expect it to strain the CI queue too much. A single job takes 12 minutes to run and should take 30-40 minutes for all jobs to complete.

Reply:
Is this new. I'm pretty sure we used to be on GitHub's free plan. Also, do the 50 runs execute in parallel? If so, that'll consume 50 of our 60 concurrent jobs

Reply:
I asked @seanlip about the GitHub plan. Looks like we're on a GitHub nonprofit org plan, which means we're on a team plan according to this [doc](https://docs.github.com/en/billing/managing-billing-for-your-github-account/discounted-subscriptions-for-github-accounts#discounts-for-nonprofits-and-libraries).

We could limit the max number of jobs running concurrently for frontend tests to 25. but if we were to run them sequentially, it would run for 12mins (per job) * 50 jobs = 600 mins (10 hours) which is not exactly ideal. By limiting the total number of concurrent runs to 25, we could complete the jobs within 30 -40 mins which I feel would be a middle ground between number of active jobs vs time taken to run

Reply:
That sounds good!

Reply:
Done